provision/core/grunt/provision.sh

```diff
@@ -0,0 +1,51 @@
+
+function install_grunt() {
+  echo " * Installing Grunt CLI"
+  npm_config_loglevel=error npm install -g grunt grunt-cli --no-optional
```

Comment:
maybe create an alias so we can simplify, like `npm-log-no-optional`

Reply:
maybe for this is better a @tomjn opinion

Reply:
If this is only used in this file then I don't think it's too important